cmd/local-gadget/main.go

```diff
@@ -19,13 +19,16 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/advise"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/audit"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/common"
 	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/containers"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/profile"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/snapshot"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/top"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/trace"
+	"github.com/inspektor-gadget/inspektor-gadget/internal/runtime/local"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+
+	// This is a blank include that actually imports all gadgets - needs to be moved into another package; TODO
```

Comment:
I think that would lead to a circular dependency (never tested with blank imports, though). I'd rather place them in an all-gadgets package that we can blank import.